providers/nrsysmond.rb

```diff
@@ -29,7 +29,7 @@
 
   r = template 'nrsysmond.cfg' do
     path      new_resource.config_file
-    cookbook  new_resource.cookbook
+    cookbook  cookbook_name
```

Comment:
Do you know when `cookbook_name` was introduced? Will this break older `chef-clients` that are in use still?